pkg/console/operator/operator.go

```diff
@@ -243,8 +243,10 @@ func (c *consoleOperator) removeConsole() error {
 	klog.V(2).Info("deleting console resources")
 	defer klog.V(2).Info("finished deleting console resources")
 	var errs []error
-	// route
-	errs = append(errs, c.routeClient.Routes(api.TargetNamespace).Delete(route.Stub().Name, &metav1.DeleteOptions{}))
+	// // route
```

Comment:
Can delete this, looks like you got the `delete` on the route for `ManagementState:Removed` on the correct controller.